3-enrich/emr-etl-runner/lib/SnowplowDashDBConn/.classpath

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you move the SnowplowDashDBConn project into this folder:

    4-storage/dashdb-loader

Thanks!